tests/e2e/e2e_test.py

```diff
@@ -1,25 +1,15 @@
 import json
-import os
-import time
 from json import JSONDecodeError
 
 import pytest
 import requests
 
 from lib.settings import SKIP_E2E_TESTS
-
-if "PRE_COMMIT" in os.environ:
```

Comment:
When running the tests from pre commit hook, I needed those lines. Are they now not needed anymore? Due to the __init__.py?

Reply:
yes, tests should now behave identical during pre_commit and normal execution!